custom_components/alexa_media/config_flow.py

```
@@ -1072,16 +1098,52 @@ async def wrapped(request, **kwargs):
+                    dict(result.headers) if hasattr(result, "headers") else "unknown",
+                )
+                return result
+            except httpx.ConnectError as ex:
                 _LOGGER.warning("Detected Connection error: %s", ex)
                 return web_response.Response(
                     headers={"content-type": "text/html"},
                     text="Connection Error! Please try refreshing. "
                     + "If this persists, please report this error to "
                     + f"<a href={ISSUE_URL}>here</a>:<br /><pre>{ex}</pre>",
                 )
+            except Exception as ex:  # pylint: disable=broad-except
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)